meerk40t/core/node/op_cut.py

```diff
@@ -108,7 +108,7 @@ def default_map(self, default_map=None):
     def drop(self, drag_node, modify=True):
         # Default routine for drag + drop for an op node - irrelevant for others...
         if drag_node.type.startswith("elem"):
-            if not drag_node.type in self._allowed_elements_dnd:
+            if not drag_node.type in self._allowed_elements_dnd or drag_node._parent.type == "branch reg":
```

Comment:
PEP8 and Codacy tend to agree that if `drag_node.type not in ....` is better. This doesn't get auto fixed by black though I did go through and change them at some point.

Reply:
Done